.travis.yml

```diff
@@ -14,8 +14,7 @@ env:
   global:
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - DOTNET_CLI_TELEMETRY_OPTOUT: 1
-mono:
-  - 4.0.5
+mono: none
```

Comment:
Yeah as of right now there's no point at all having Mono be installed because we don't running anything on it anyway. It was just for sake. When the time comes and we have a real objective, we'll need to make sure we test in the actual Mono-related scenarios we care about (as opposed to some random build of Mono).